### PR TITLE
Extend stop1 acceptance test

### PR DESCRIPTION
**Pull request by martiniil**
The new tests verify that #93 is solved.

### CODE REVIEW

File: prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md [hunk 1]
```diff
@@ -54,3 +54,14 @@ These acceptance tests check that the real robot system reacts to RUN_PERMITTED
 ### Expected Results
   1. Same as Test 1
   2. Robot should stop immediatly and smoothly.
+
+## 4. Fast RUN_PERMITTED switching
+### Test Sequence
+  1. Same as Test 1
+  2. Perform as fast as you can: Release and press acknowledge button. Repeat several times.
```
Comment (SansoneG) [inline prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md]: I don't see how Point 2 and 3 are different.

Comment (martiniil) [reply prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md]: The order is different. 2. is about the sequence release->press and 3. the other way around.

Comment (SansoneG) [reply prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md]: I think I understand now. The wording might be confusing. I interpreted "repeat several times" as in "keep on pressing and releasing the button as fast as you can", but I now think it is meant as "repeat the single action of a fast release+press multiple times no matter how much time passes in between the single actions."

Comment (martiniil) [reply prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md]: Is it better now?

**Review by SansoneG — APPROVED** [review]

File: prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md (hunk 1 above)
Comment (SansoneG) [reply prbt_hardware_support/test/acceptancetests/acceptancetest_stop1.md]: Yes, thank you!

**Pull request merged**